views/js/qtiCommonRenderer/renderers/interactions/UploadInteraction.js

```diff
@@ -29,12 +29,13 @@ define([
     'tpl!taoQtiItem/qtiCommonRenderer/tpl/interactions/uploadInteraction',
     'taoQtiItem/qtiCommonRenderer/helpers/container',
     'taoQtiItem/qtiCommonRenderer/helpers/instructions/instructionManager',
+    'taoQtiItem/qtiCreator/helper/uploadMime',
```

Comment:
oops, qtiCommonRenderer should not depends on qtiCreator. It should be the over way around. Qti Creator depends on Qti Common Renderer. If it is too troublesome, you may leave it aside for now.